src/internal-api.yml

```
@@ -200,6 +200,12 @@ paths:
                     type: object
                     nullable: true
                     additionalProperties: true
+                  limit:
+                    type: integer
+                    nullable: true
+                  skip:
+                    type: integer
+                    nullable: true
```

Define them in an options property

```suggestion
                  options:
                    type: object
                    properties:
                      limit:
                        type: integer
                        nullable: true
                      skip:
                        type: integer
                        nullable: true
```